scripts/test/test-with-eksctl.sh

```diff
@@ -268,6 +268,10 @@ verify_controller_has_lease
 # the node for ENI Trunking/Branching
 set_pod_eni_flag_on_ipamd "true"
 
+# Allow for IPAMD to label the node after startup
+# TODO: Handle this in the Test Suite in more concrete manner
```

Comment:
Yes, this is hacky for now. IPAMD on startup sets the Label on the Nodes `vpc.amazonaws.com/has-trunk-attached` this allows for the controller to start managing the nodes. 

In our new Ginko test suite, we have one test that expects all the Node to have this label otherwise fails. IPAMD, takes some time to add this label and if that test is first to run then the suite fails. Will fix this properly soon.